src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -300,6 +301,11 @@
               <FutureEffective :filing=filing class="mt-6" />
             </template>
 
+            <template v-else-if="filing.isTypeLimitedRestoration"
+            >
+              <LimitedRestorationFiling :filing="filing" class="mt-6" />
+            </template>
```

Comment:
This looks good.

The rule for where to put this (since order matters in this long if-then-else) is "as low as possible" so it doesn't pre-empt something it shouldn't but gets pre-empted by something more important.

Reply:
Thanks!